src/core/worker.c

```diff
@@ -118,8 +118,13 @@ QuicWorkerInitialize(
             break;
         }
 
-        if (MsQuicLib.ExecutionConfig && MsQuicLib.ExecutionConfig->Flags & QUIC_EXECUTION_CONFIG_FLAG_HIGH_PRIORITY) {
-            ThreadFlags |= CXPLAT_THREAD_FLAG_HIGH_PRIORITY;
+        if (MsQuicLib.ExecutionConfig) {
+            if (MsQuicLib.ExecutionConfig->Flags & QUIC_EXECUTION_CONFIG_FLAG_HIGH_PRIORITY) {
+                ThreadFlags |= CXPLAT_THREAD_FLAG_HIGH_PRIORITY;
+            }
+            if (MsQuicLib.ExecutionConfig->Flags & QUIC_EXECUTION_CONFIG_FLAG_AFFINITIZE) {
+                ThreadFlags |= CXPLAT_THREAD_FLAG_SET_AFFINITIZE;
```

Comment:
Looks like just adding flag? where is the actual logic?

Reply:
The logic was already implemented elsewhere. This is just a new config flag.